tests/contest/runtimetest/src/tests.rs

```diff
@@ -550,6 +551,36 @@ pub fn test_io_priority_class(spec: &Spec, io_priority_class: IOPriorityClass) {
     }
 }
 
+pub fn validate_process(spec: &Spec) {
+    let process = spec.process().as_ref().unwrap();
+
+    if process.cwd().ne(&getcwd().unwrap()) {
+        eprintln!(
+            "error due to spec cwd want {:?}, got {:?}",
+            process.cwd(),
+            getcwd().unwrap()
+        )
+    }
+
+    for env_str in process.env().as_ref().unwrap().iter() {
+        match env_str.split_once("=") {
+            Some((env_key, env_val)) => {
+                if env::var(env_key).unwrap() != env_val {
+                    eprintln!(
+                        "error due to spec environment value of {:?} want {:?}, got {:?}",
+                        env_key,
+                        env::var(env_key).unwrap(),
+                        env_val
+                    )
```

Comment:
```suggestion
            Some((env_key, expected_val)) => {
                let actual_val = env::var(env_key).unwrap();
                if  actual_val != expected_val {
                    eprintln!(
                        "error due to mismatch of environment variable {:?} : want value {:?}, got {:?}",
                        env_key,
                        actual_val,
                        expected_val
                    )
```

Main change is to extract it as a variable, you can keep the error message same if you want.

Reply:
I fixed this.
https://github.com/youki-dev/youki/pull/2968/commits/66a404ed709b82fe3618745f09dd3be2c0fde715